<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>778</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::DashLine</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="filterToggleButton">
      <property name="text">
       <string>Filter</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="filterGroupBox">
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="presetsGroupBox">
           <property name="title">
            <string>Presets</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="overstockedRadioButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Overstocked</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="trendingRadioButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Trending</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="customRadioButton">
              <property name="text">
               <string>Custom</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="filterButton">
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="boughtGroupBox">
           <property name="title">
            <string>How many books have been bought?</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QGroupBox" name="boughtMoreThanBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>More than</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QSlider" name="boughtMoreThenSlider">
                 <property name="maximum">
                  <number>20</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="boughtMoreThenSpin">
                 <property name="maximum">
                  <number>20</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="boughtLessThanBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Less than</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QSlider" name="boughtLessThenSlider">
                 <property name="maximum">
                  <number>20</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="boughtLessThenSpin">
                 <property name="maximum">
                  <number>20</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="stockGroupBox">
           <property name="title">
            <string>How many books there are in stock?</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QGroupBox" name="instockMoreThanBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>More than</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QSlider" name="instockMoreThenSlider">
                 <property name="maximum">
                  <number>20</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="instockMoreThenSpin">
                 <property name="maximum">
                  <number>20</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="instockLessThanBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Less than</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QSlider" name="instockLessThenSlider">
                 <property name="maximum">
                  <number>20</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="instockLessThenSpin">
                 <property name="maximum">
                  <number>20</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionReconnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReconnect">
   <property name="text">
    <string>Reconnect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>boughtLessThenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>boughtLessThenSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boughtLessThenSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>boughtLessThenSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boughtMoreThenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>boughtMoreThenSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boughtMoreThenSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>boughtMoreThenSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>instockLessThenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>instockLessThenSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>instockLessThenSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>instockLessThenSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>instockMoreThenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>instockMoreThenSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>instockMoreThenSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>instockMoreThenSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>694</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterToggleButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
